src/fs/utils.rs

```diff
@@ -277,3 +277,46 @@ pub fn add_motionlist_patch<P: AsRef<Path>, Q: AsRef<Path>>(tree: &mut Tree<ApiL
     error!("Could not add file {} to API tree. Reason: This is not a motion_list.bin file.", full_path.display());
     None
 }
+
+pub fn add_bgm_property_patch<P: AsRef<Path>, Q: AsRef<Path>>(tree: &mut Tree<ApiLoader>, phys_root: P, local: Q) -> Option<Hash40> {
+    let local = local.as_ref();
+    let base_local = local.with_extension("bin");
```

Comment:
You sure about that? I'd assume there are more files with a .bin extension than bgm_property?

Reply:
messaged you about this already, but with_local just sets the extension to the extension given, so i'm not sure what you mean by this.